src/main.ts

```diff
@@ -128,7 +128,11 @@ function fileWatch(config: webpack.Configuration, args: any, app?: express.Appli
 				reject(err);
 			}
 			if (stats) {
-				const runningMessage = args.serve ? `Listening on port ${args.port}` : 'watching...';
+				const runningMessage = args.serve
+					? `Listening on port ${
+							args.port
+					  }\nPlease note the serve option is not intended to be used to serve applications in production.`
```

Comment:
Should we colour this at all? But otherwise :+1:

Reply:
I hope that's not needed, but I was a little surprised we needed to add the message. How about we merge this and if there's still a need we can make the message more noticeable?

Reply:
Works for me